src/ZplBuilder.php

```diff
@@ -88,6 +88,27 @@ public function setFont(string $font, float $size, ?float $width = null): void
         $this->commands[] = $command;
     }
 
+    public function setQuantity(int $quantity, int $pauseQty = 0, int $replicate = 1): void
+    {
+        $this->commands[] = '^PQ' . $quantity . ',' . $pauseQty . ',' . ($replicate <= 0 ? 1 : $replicate);
+    }
```

Comment:
@erikn69 is there a reason for the `$replicate` default to be 1? The ZPL manual says its 0

<img width="936" height="444" alt="Image" src="https://github.com/user-attachments/assets/1b309ea5-16c4-4d6b-a7fe-7a59f3f0aa2a" />

```suggestion
    public function setQuantity(int $quantity, int $pauseQty = 0, int $replicate = 0): void
    {
        $this->commands[] = '^PQ' . $quantity . ',' . $pauseQty . ',' . ($replicate < 0 ? 0 : $replicate);
    }
```